engines/griffon/cutscenes.cpp

```diff
@@ -133,6 +139,27 @@ const char *story2[27] = {
 	"and I am free to die as I please."
 };
 
+#ifdef USE_TTS
+int textToSpeech(int nextparagraph, const char *story[], int arraysize) { 
+Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+if (_ttsMan != nullptr && ConfMan.getBool("tts_enabled")) {
+	if ((_ttsMan->isSpeaking()) == false) {
+		Common::String paragraph;
+		while (nextparagraph < arraysize && story[nextparagraph][0] != ' ') {
```

Comment:
You could also check that `story[nextparagraph][0] != 0` to handle the empty lines in the first story array.